app/middleware/rate_limiter.py

```
@@ -0,0 +1,216 @@
+"""Rate limiting middleware with Redis backend."""
+
+import os
+import redis
+from flask import request, jsonify, g
```

Ruff (0.11.9)

`flask.jsonify` imported but unused

Remove unused import

(F401)